docs/docs/configuration/db.md

```diff
@@ -78,8 +78,8 @@ $ trivy image --java-db-repository registry.gitlab.com/gitlab-org/security-produ
     `java-db-registry:latest` => `java-db-registry:latest`, but `java-db-registry` => `java-db-registry:1`.
 
 ## Remove DBs
-The `--reset` flag removes all caches and databases.
+"trivy clean" command removes caches and databases.
 
 ```
-$ trivy image --reset
+$ trivy clean --all
```

Comment:
We can use separate flags to remove DBs:
```suggestion
$ trivy clean --vuln-db --java-db
2024-06-24T11:42:31+06:00       INFO    Removing vulnerability database...
2024-06-24T11:42:31+06:00       INFO    Removing Java database...
```

Reply:
Similar to https://github.com/aquasecurity/trivy/pull/6993#discussion_r1650411084, `--reset` is not just removing databases, but removing all caches.

Reply:
right. But this is DB page.
I think  this page should say how to remove only DBs (instead of removing all caches).

Reply:
Oh, yes. You're right. I just replaced the existing flag (`--reset`) with the equivalent one (`--all`), but we now have better flags to remove DBs. Thanks.